modules/n1ql/examples/n1ql-language-reference/udf-locations-test.n1ql

```diff
@@ -0,0 +1 @@
+EXECUTE FUNCTION locations("see");
```

Comment:
Nitpick: "See" in the context of a location confused me in these examples.

After some thought, I came to https://en.wikipedia.org/wiki/Episcopal_see and then finally "sightseeing" as it's an "activity" word. Would something like "visit" be clearer as an activity?

Reply:
Beyond our control unfortunately 😄

Reply:
Although I could choose a different activity!